development/development.env

```diff
@@ -110,3 +110,6 @@ IPFABRIC_SSL_VERIFY="True"
 IPFABRIC_TIMEOUT=15
 
 NAUTOBOT_SSOT_ENABLE_ITENTIAL="True"
+
+NAUTOBOT_SSOT_ENABLE_SLURPIT="True"
```

Comment:
This should be False.

Reply:
@lpconsulting321 That is most likely being caused due to your integration being enabled. Making this change should resolve the issue.

Reply:
<3